internal/conf/configuration.go

```diff
@@ -28,7 +28,7 @@ var postgresNamesRegexp = regexp.MustCompile(`^[a-zA-Z_][a-zA-Z0-9_]{0,62}$`)
 // So this 4 * Math.ceil(24/3) = 32 and 4 * Math.ceil(64/3) = 88 for symmetric secrets
 // Since Ed25519 key is 32 bytes so we have 4 * Math.ceil(32/3) = 44
 var symmetricSecretFormat = regexp.MustCompile(`^v1,whsec_[A-Za-z0-9+/=]{32,88}`)
-var asymmetricSecretFormat = regexp.MustCompile(`^v1a,whpk_[A-Za-z0-9+/=]{44,};whsk_[A-Za-z0-9+/=]{44,}$`)
+var asymmetricSecretFormat = regexp.MustCompile(`^v1a,whpk_[A-Za-z0-9+/=]{44,}:whsk_[A-Za-z0-9+/=]{44,}$`)
```

Comment:
`[A-Za-z0-9+/=]` this doesn't look like all of Base64 to me...

Reply:
oh what's missing? As I understand it should be 

Lowercase: a-z (26)
Uppercase: A-Z (26)
Numbers: 0-9 (10)
Special Characters: +/ (2)
So 26 + 26 + 10 + 2 = 64

Reply:
Resolving as discussed during catchup